src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -146,7 +146,7 @@ public static bool ShowContainerPickerWindow(IntPtr hwnd, out string connectionS
             bool? dialogResult = dialog.ShowModal();
             if (dialogResult.GetValueOrDefault(false))
             {
-                connectionString = dialog.SelectedContainerConnectionString;
+                connectionString = dialog.Model.SelectedContainerConnectionString;
```

Comment:
>Model.SelectedContainerConnectionString [](start = 42, length = 39)

nit: This feels pretty weird. Does the view model need to be public or could you just keep the SelectedContainerConnectionString property and return the view models value?